pxc/docker/run-test57

```diff
@@ -22,15 +22,19 @@ docker run --rm \
     export MTR_ARGS='${MTR_ARGS}'
     export MTR_REPEAT='${MTR_REPEAT}'
     export MTR_VAULT_TOKEN='${MTR_VAULT_TOKEN}'
+	export FULL_MTR='${FULL_MTR}'
+	export GALERA_PARALLEL_RUN='${GALERA_PARALLEL_RUN}'
+	export MTR_SUITES='${MTR_SUITES}'
+	export PARALLEL_RUN='${PARALLEL_RUN}'
 
     mkdir /tmp/results
     cp /tmp/pxc/results/*.tar.gz /tmp/results
 
     bash -x /tmp/scripts/test-binary-pxc57 /tmp/results
 
     mkdir /tmp/results/log
-    rsync -a --prune-empty-dirs --include '*/' --include '*.err' --exclude '*' /tmp/results/PXC/mysql-test/var /tmp/results/log
-    sudo tar -C /tmp/results/log -czf "pxc57-test-mtr_logs.tar.gz" var
+    rsync -a --prune-empty-dirs --include '*/' --include '*.err' --exclude '*' /tmp/results/PXC/mysql-test/mtr_var /tmp/results/log
+    sudo tar -C /tmp/results/log -czf "pxc57-test-mtr_logs.tar.gz" mtr_var || true
```

Comment:
Why isn't it an error if we can't create an archive? Please add a comment about it, as it's not trivial

Reply:
It was added for testing purposes, removed `|| true` from archive creation.